x/superfluid/keeper/msg_server_test.go

```diff
@@ -192,3 +193,191 @@ func (suite *KeeperTestSuite) TestMsgLockAndSuperfluidDelegate() {
 		}
 	}
 }
+
+// TestMsgSuperfluidDelegate_Event tests that events are correctly emitted
+// when calling SuperfluidDelegate.
+func (suite *KeeperTestSuite) TestMsgSuperfluidDelegate_Event() {
```

Comment:
I noticed that events tests are very identical to their actual tests and there is repetitive code like (`TestMsgSuperfluidDelegate` and `TestMsgSuperfluidDelegate_Event`) 

is there any way we can just use one?